### ISSUE TITLE
invocation of `super` silently fails

### ISSUE THREAD

**Issue by Olipro** [ayecue/greybel-vs#136]
Given:

```Lua
Foo = {}
Foo.f = function; return 123; end function
Bar = new Foo
Bar.f = function; return super.f; end function
print Bar.f
print """" + Bar.f + """"
```

The first print silently fails.

The second one reveals what I imagine is an internal interpreter error: `Runtime error: Cannot read properties of undefined (reading 'toString')`

**Comment by ayecue** [comment]
Should be fixed now with 1.9.16 - https://github.com/ayecue/greybel-vs/pull/138 merged. Main issue was that super didn't actually use the owning map but instead the call context as reference which pretty much lead to various issues beside of this one. Anyway should be fixed now.

**Comment by Olipro** [comment]
All good!
